package/AUTHORS

```diff
@@ -209,6 +209,7 @@ Chronological list of authors
   - Vishal Parmar
   - Moritz Schaeffler
   - Xu Hong Chen
+  - Zhiyi Wu
```

Comment:
```suggestion
```

@xiki-tempula, seem to be already under authors in 2017 (assuming it is you, since you have been contributing for a while).

Reply:
I think @IAlibay was about adding yourself to the list of authors in the `CHANGELOG`, for the `2.5.0` release.

Reply:
@RMeli  Yes, that is me. Shall I remove it?

Reply:
I see. I have changed it.